src/request_items.rs

```diff
@@ -228,10 +239,16 @@ impl RequestItems {
         let mut text_fields = Vec::<(String, String)>::new();
         for item in self.0 {
             match item {
-                RequestItem::JsonField(_, _) => {
+                RequestItem::JsonField(..) | RequestItem::JsonFieldFromFile(..) => {
                     return Err(anyhow!("JSON values are not supported in Form fields"));
                 }
                 RequestItem::DataField(key, value) => text_fields.push((key, value)),
+                RequestItem::DataFieldFromFile(key, value) => {
+                    let mut file = File::open(value)?;
+                    let mut contents = String::new();
+                    file.read_to_string(&mut contents)?;
+                    text_fields.push((key, contents));
+                }
                 RequestItem::FormFile(..) => unreachable!(),
                 _ => {}
```

Comment:
I think it's better to not use wildcards for request items anywhere in the codebase any more, so we know we didn't forget anything.

[`to_curl.rs`](https://github.com/ducaale/xh/blob/90dbf65333746845d3c5f21f536e9386ef7e40aa/src/to_curl.rs#L271) is wrong right now. There might be others.